cfme/cloud/provider.py

```diff
@@ -69,6 +69,8 @@
             "5.5": AngularSelect("provider_region")}),
         ("api_version", AngularSelect("api_version")),
         ('infra_provider', AngularSelect("provider_id")),
+        ('google_region_select', AngularSelect("provider_region")),
```

Comment:
Since this element is the same as the amazon one above, I wonder if we can combine them and then just attack the form_mappings on the Provider classes to make the change fixed throughout, I dont' like having the same element referred to by two different names if we don't have to.